docs/designers-developers/key-concepts.md

```diff
@@ -20,7 +68,7 @@ Additionally, how do we even know this came from our editor? Maybe someone snuck
 
 ## The post dichotomy
 
-A Gutenberg post is the proper block-aware representation of a post, a collection of semantically consistent descriptions of what each block is and what its essential data is. This representation only ever exists in memory. It is the [chase](https://en.wikipedia.org/wiki/Chase_(printing)) in the typesetter's workshop, ever-shifting as sorts are attached and repositioned.
+A Gutenberg post is the proper block-aware representation of a post, a collection of semantically consistent descriptions of what each block is and what its essential data is. This representation only ever exists in memory. It is the [chase](<https://en.wikipedia.org/wiki/Chase_(printing)>) in the typesetter's workshop, ever-shifting as sorts are attached and repositioned.
```

Comment:
Is this a particular Markdown quirk? I've never seen `[text](<url>)`.

Reply:
I honestly hadn't seen it before either (@0aveRyan what were you using to edit your markdown?) but I think it's possibly due to the fact that the URL contains parentheses. Happy to revert it.

Reply:
GitHub is visibly fine with it, so the remaining concern is whether it would break in the .org handbook. If it's supported, then I have no objection. :)

Reply:
Sadly I don't have access to a dev environment for the handbook right now — the files are available but not a SQL for presetting the environment — so I'll revert this and we can revisit later if needed :)

Reply:
oh apologies! i don't have anything too wild in my VS Code setup 🤔 aimed for more reorg and addition/intro in this first pr.